src/management/metadata.ts

```diff
@@ -0,0 +1,46 @@
+export interface FirebaseProjectMetadata {
+  name: string /* The fully qualified resource name of the Firebase project */;
+  projectId: string;
+  projectNumber: string;
+  displayName: string;
+  resources: DefaultResource;
+}
+
+export interface DefaultResource {
+  hostingSite: string;
+  realtimeDatabaseInstance: string;
+  storageBucket: string;
+  locationId: string;
+}
+
+export interface AppMetadata {
+  name: string /* The fully qualified resource name of the Firebase App*/;
```

Comment:
space after `App`

Reply:
Updated in https://github.com/firebase/firebase-tools/pull/1470